lib/ProductOpener/DataQualityFood.pm

```diff
@@ -1319,25 +1319,31 @@ sub check_nutrition_data ($product_ref) {
 		}
 
 		# sum of nutriments that compose fiber can not be greater than the value of fiber
-		if (
-			(defined $product_ref->{nutriments}{fiber_100g})
-			and (
-				(
-					(
-						(defined $product_ref->{nutriments}{'soluble-fiber_100g'})
-						? $product_ref->{nutriments}{'soluble-fiber_100g'}
-						: 0
-					) + (
-						(defined $product_ref->{nutriments}{'insoluble-fiber_100g'})
-						? $product_ref->{nutriments}{'insoluble-fiber_100g'}
-						: 0
+		if (defined $product_ref->{nutriments}{fiber_100g}) {
+			my $soluble_fiber
+				= defined $product_ref->{nutriments}{'soluble-fiber_100g'}
+				? $product_ref->{nutriments}{'soluble-fiber_100g'}
+				: 0;
+			my $insoluble_fiber
+				= defined $product_ref->{nutriments}{'insoluble-fiber_100g'}
+				? $product_ref->{nutriments}{'insoluble-fiber_100g'}
+				: 0;
+			my $total_fiber = $soluble_fiber + $insoluble_fiber;
+
+			if ($total_fiber > $product_ref->{nutriments}{fiber_100g} + 0.001) {
+				# ignore if there is "<" symbol (example: <1 + 5 = 5, issue #11075)
```

Comment:
I think it would be better to add $soluble_fiber and $insoluble_fiber to $total_fiber only if there isn't a corresponding < modifier.

That way we will trigger the warning if for instance we have 5 g fiber, 10 g insoluble fiber, and < 1g soluble fiber).

Reply:
Good idea. I applied these changes. If you could review it again, just to be sure. Thanks

Reply:
@stephanegigandet, forgot to push 1 file